.travis.yml

```diff
@@ -1,24 +1,43 @@
 language: cpp
 
-# We need to use sudo because of Docker
-sudo: require
+# Use the faster container-based infrastructure.
+sudo: false
+
+# Use Trusty beta build environment.
+dist: trusty
+
+addons:
+  apt:
+    packages:
+    - libxml2-dev
+    - flex
+    - bison
+    - pkg-config
+    # Optional dependency needed by hwtest
+    - libpciaccess-dev
+    # Optional dependency needed by demmt
+    - libdrm-nouveau2
+   #- libseccomp-dev  # Uncomment if Travis-CI includes on ubuntu-trusty apt package whitelist
+    # Optional dependency needed by nva
+    - libpciaccess-dev
+    - libx11-dev
+    - libxext-dev
+    - python3
```

Comment:
This should be `python3-dev` for the reason outlined in https://github.com/envytools/envytools/pull/118/commits/265d4f1ab73763b5924bfd0989dbb3b41489645a